frontend/appflowy_flutter/packages/appflowy_editor/lib/src/render/image/image_upload_widget.dart

```diff
@@ -98,16 +166,50 @@ class _ImageUploadMenuState extends State<ImageUploadMenu> {
             color: Colors.black.withOpacity(0.1),
           ),
         ],
-        // borderRadius: BorderRadius.circular(6.0),
+        borderRadius: BorderRadius.circular(6.0),
       ),
       child: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          _buildHeader(context),
-          const SizedBox(height: 16.0),
-          _buildInput(),
+        children: <Widget>[
+          TabBar(
+            controller: _tabController,
+            tabs: const [
+              Text(
+                'Upload Image',
+                textAlign: TextAlign.left,
+                style: TextStyle(
+                    fontSize: 14.0,
+                    color: Colors.black,
+                    fontWeight: FontWeight.w500),
+              ),
+              Text(
+                'URL Image',
+                textAlign: TextAlign.left,
+                style: TextStyle(
+                    fontSize: 14.0,
+                    color: Colors.black,
+                    fontWeight: FontWeight.w500),
+              )
+            ],
+          ),
+          SizedBox(
+              height: 200.0,
+              child: TabBarView(controller: _tabController, children: <Widget>[
+                InkWell(
+                  onTap: (() => _pickFiles()),
```

Comment:
Redundant paranthesis.
```suggestion
                  onTap: _pickFiles,
```